frontend/packages/dev-console/console-extensions.json

```diff
@@ -725,6 +740,24 @@
       "required": ["OPENSHIFT_BUILDCONFIG"]
     }
   },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "developer-catalog",
+      "perspective": "admin",
+      "section": "home",
+      "name": "%devconsole~Developer Catalog%",
+      "href": "/catalog",
```

Comment:
Tested the PR. Everything looks fine but one small doubt, when we had meeting with @kevinhatchoua, I had the impression that he informed to keep "Software Catalog" in both the perspectives. 

@kevinhatchoua , if you confirm on this, we can merge this PR.